qp_qiime2/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .qiime2 import rarefy
+
+__all__ = ['qiime2']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'qiime2', '4.2017', 'QIIME 2')
```

Comment:
Is it possible to import the version from qiime2 directly, so we don't have to manually update it? This way it will work with whatever version of q2 is installed.